great_expectations/self_check/util.py

```diff
@@ -3000,6 +3003,17 @@ def check_json_test_result(  # noqa: C901 - 52
         result = result.to_json_dict()
         for key, value in test["output"].items():
             # Apply our great expectations-specific test logic
+            if v2_or_v3_api is "v2" and key in [
```

Comment:
these metrics are not implemented for V2 `MultiColumnMapExpectations`

Reply:
no longer having to do this split after adding the `pandas_v2_api` and `pandas_v3_api` tests :)